tfexec/options.go

```diff
@@ -40,6 +32,11 @@ func Backup(path string) *BackupOption {
 	return &BackupOption{path}
 }
 
+// DisableBackup is a convenience method for Backup("-"), indicating backup state should be disabled.
+func DisableBackup() *BackupOption {
```

Comment:
Happy with this direction - we can provide more convenience options as use cases become clear.